pwndbg/abi.py

```diff
@@ -111,3 +116,31 @@ class SigreturnABI(SyscallABI):
 linux_i386_srop = ABI(['eax'], 4, 0)
 linux_amd64_srop = ABI(['rax'], 4, 0)
 linux_arm_srop = ABI(['r7'], 4, 0)
+
+
+# See issue #264 for more reference
+# 'GNU/Linux' for linux
+# 'none' for bare metal
+abi = None
+
+def is_linux():
```

Comment:
I'd rather just have a global that we update whenever `abi` is updated, so we can check:

`if pwndbg.abi.linux` vs `if pwndbg.abi.is_linux()`